hoot-core/src/main/cpp/hoot/core/algorithms/WayJoiner2.cpp

```diff
@@ -525,78 +371,6 @@ void WayJoiner2::_rejoinSiblings(deque<long>& way_ids)
   }
 }
 
-void WayJoiner2::_determineKeeperFeature(WayPtr parent, WayPtr child, WayPtr& keeper,
-                                         WayPtr& toRemove)
-{
-  // this is kind of a mess
-
-  const QString tagMergerClassName = ConfigOptions().getTagMergerDefault();
-  LOG_VART(tagMergerClassName);
-  if (parent->getStatus() == Status::Unknown1)
-  {
-    if (tagMergerClassName == "hoot::OverwriteTagMerger" ||
-        tagMergerClassName == "hoot::OverwriteTag2Merger")
-    {
-      keeper = child;
-      toRemove = parent;
-    }
-    else if (tagMergerClassName == "hoot::OverwriteTag1Merger")
-    {
-      keeper = parent;
-      toRemove = child;
-    }
-    else
-    {
-      keeper = parent;
-      toRemove = child;
-    }
-  }
-  else if (child->getStatus() == Status::Unknown1 ||
-           (parent->getStatus() == Status::Conflated && child->getStatus() == Status::Conflated))
-  {
-    if (tagMergerClassName == "hoot::OverwriteTagMerger" ||
-        tagMergerClassName == "hoot::OverwriteTag2Merger")
-    {
-      keeper = parent;
-      toRemove = child;
-    }
-    else if (tagMergerClassName == "hoot::OverwriteTag1Merger")
-    {
-      keeper = child;
-      toRemove = parent;
-    }
-    else
-    {
-      keeper = parent;
-      toRemove = child;
-    }
-  }
-  // does this make sense??
-  else
-  {
-    keeper = parent;
-    toRemove = child;
-  }
-}
-
-bool WayJoiner2::_handleOneWayStreetReversal(WayPtr wayWithTagsToKeep,
-                                             ConstWayPtr wayWithTagsToLose)
-{
-  OneWayCriterion oneWayCrit;
-  if (oneWayCrit.isSatisfied(wayWithTagsToLose) &&
-      !oneWayCrit.isSatisfied(wayWithTagsToKeep) &&
-      // note the use of an alternative isSimilarDirection method
-      !DirectionFinder::isSimilarDirection2(
-        _map->shared_from_this(), wayWithTagsToKeep, wayWithTagsToLose))
-  {
-    LOG_TRACE("Reversing order of " << wayWithTagsToKeep->getElementId());
-    // make sure this reversal gets done before checking the join type later on
-    wayWithTagsToKeep->reverseOrder();
-    return true;
-  }
-  return false;
-}
-
```

Comment:
This function is just moved to be in the same order it is found within the class declaration.